test_scripts/API/VehicleData/StabilityControlsStatus/008_GetVehicleData_StabilityControlsStatusNotAllowedByPolicy.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check processing of disallowed by policy stabilityControlsStatus data
+-- via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed, stabilityControlsStatus NOT allowed by Policies
+--
+-- Steps:
+-- 1) App sends GetVehicleData in (NONE, FULL, LIMITED, BACKGRAUND) level (with stabilityControlsStatus = true) request to SDL
+--  - SDL not sends VehicleInfo.GetVehicleData (with stabilityControlsStatus = true) request to HMI
+--  - SDL sends VehicleInfo.GetVehicleData response ( resultCode = DISALLOWED, success = false )
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+local runner = require('user_modules/script_runner')
+
+--[[ Local Variables ]]
+local resultCode = "DISALLOWED"
+
+--[[ Local Functions ]]
+function ptUpdateMin(pTbl)
+  pTbl.policy_table.app_policies[common.getConfigAppParams().fullAppID].groups = { "Base-4", "Emergency-1" }
+  local grp = pTbl.policy_table.functional_groupings["Emergency-1"]
+  grp.rpcs.GetVehicleData.hmi_levels = {
+    "FULL",
+    "LIMITED",
+    "BACKGROUND",
+    "NONE" 
+  }
+  grp.rpcs.GetVehicleData.parameters = common.EMPTY_ARRAY
+  pTbl.policy_table.vehicle_data = nil
+end
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, init HMI, connect default mobile", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("PTU", common.policyTableUpdate, { ptUpdateMin })
+
+common.Title("Test")
+common.Step("Send get vehicle data, NONE level", common.processGetVDunsuccess, { "stabilityControlsStatus", resultCode })
+common.Step("App activate", common.activateApp)
+common.Step("Send get vehicle data, FULL level", common.processGetVDunsuccess, { "stabilityControlsStatus", resultCode })
+common.Step("Set HMI Level to Limited", common.hmiLeveltoLimited, { 1, "MAIN" })
+runner.Step("Send get vehicle data, LIMITED level", common.processGetVDunsuccess, { "stabilityControlsStatus", resultCode })
+runner.Step("Set HMI Level to BACKGROUND", common.hmiLeveltoBackground)
+runner.Step("Send get vehicle data, BACKGROUND level", common.processGetVDunsuccess, { "stabilityControlsStatus", resultCode })
```

Comment:
@ydementieiev @aderiabin @VjKlepikov What do you say about the usage of world-known abbreviations in description and text? 
- BACKGROUND - **BKGND**
- LIMITED - **LTD**
NONE & FULL is quite short, so no abbreviations are required

Reply:
@AByzhynar  in sdl_aft_tests_scripts don't used abbreviations for them yet, but we can change to abbreviations if this will be more readable,@aderiabin @VjKlepikov I what do you think?

Reply:
@AByzhynar Step descriptions goal is to explain action and checks which performed in the step to everyone who will read a script's scenario. The world-known standard abbreviations are not intuitive as for me. So I am not agree to use them.